packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -1,44 +1,55 @@
 import { FormPoster } from '@bigcommerce/form-poster';
 
+import { BillingAddressActionCreator } from '../../../billing/';
 import { CheckoutActionCreator, CheckoutStore } from '../../../checkout';
-import { InvalidArgumentError, MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
-import { ApproveDataOptions, ButtonsOptions, ClickDataOptions, PaypalCommercePaymentProcessor } from '../../../payment/strategies/paypal-commerce';
+import { InvalidArgumentError, MissingDataError, MissingDataErrorType, RequestError } from '../../../common/error/errors';
+import { OrderActionCreator } from '../../../order';
+import { PaymentActionCreator } from '../../../payment/';
+import { ApproveActions, ApproveDataOptions, ButtonsOptions, ClickDataOptions, CurrentShippingAddress, OnCancelData, PayPalCommercePaymentMethod, PaypalCommercePaymentProcessor, ShippingAddress, ShippingChangeData, ShippingData } from '../../../payment/strategies/paypal-commerce';
+import { ConsignmentActionCreator, ConsignmentLineItem } from '../../../shipping';
 import { CheckoutButtonInitializeOptions } from '../../checkout-button-options';
 import CheckoutButtonStrategy from '../checkout-button-strategy';
 
 export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrategy {
+    private _paymentMethod?: PayPalCommercePaymentMethod;
     private _isCredit?: boolean;
-    private _isVenmo?: boolean;
+    private _currentShippingAddress?: CurrentShippingAddress;
     private _isVenmoEnabled?: boolean;
+    private _isVenmo?: boolean;
+    private _shippingData?: ShippingData;
```

Comment:
We should avoid adding this variable because we already have `_currentShippingAddress` what has the same values inside.

Reply:
removed